srcpkgs/encrypted-dns/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'encrypted-dns'
+pkgname=encrypted-dns
+version=0.9.13
+revision=1
+build_style=cargo
+makedepends="libsodium-devel"
+short_desc="DNSCrypt v2 server with support for DNSSEC and DoH forwarding"
+maintainer="MeganerdNL <meganerd@meganerd.nl>"
+license="MIT"
+homepage="https://github.com/DNSCrypt/encrypted-dns-server"
+distfiles="https://github.com/DNSCrypt/encrypted-dns-server/archive/refs/tags/${version}.tar.gz"
+checksum=5e4f9143313bf58888c31ec4e220e9fb65b28b60fe5b6aff872f9f2ecb7537d5
+system_accounts="_encrypted_dns"
+_encrypted_dns_homedir="/var/lib/encrypted-dns"
+mutable_files="/var/lib/encrypted-dns/undelegated.txt"
+
+if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
+	XBPS_CROSS_RUSTFLAGS+=" -latomic"
+	makedepends+=" libatomic-devel"
+fi
+
+case "$XBPS_TARGET_MACHINE" in
+	*-musl) makedepends+=" musl-devel libstdc++";;
+esac
+
+post_install() {
+	vlicense LICENSE
+	vinstall undelegated.txt 0644 var/lib/encrypted-dns
+	vsconf example-encrypted-dns.toml
+	vmkdir etc/encrypted-dns
+	touch ${DESTDIR}/etc/encrypted-dns/.placeholder
```

Comment:
```suggestion
```
the user can create the dir themselves (and `make_dirs=` is the proper way to do this anyways)